src/main/java/games/strategy/triplea/delegate/AbstractMoveExtendedDelegateState.java

```diff
@@ -6,8 +6,7 @@
 class AbstractMoveExtendedDelegateState implements Serializable {
   private static final long serialVersionUID = -4072966724295569322L;
   Serializable superState;
-  // add other variables here:
-  public boolean m_nonCombat;
```

Comment:
So, if this field is removed, and a new save game is generated and sent to an older client, the old client will always end up with a value of `false` for this field.  That may be ok.

However, if an older client happens to use this field and stores a `true` value, then round-tripping the save game between this client and an older client won't preserve the `true` value.  You'd probably have to go back to at least 3635 and verify this field isn't being used there either.  Or make life easy on yourself and not remove serializable fields until the next incompatible release. :smile:

Reply:
After fiddling around with gradle, different dependencies and a different source folder structure for at least 10 minutes, I can verify this field isn't used in 3635